### ISSUE TITLE
Suggestion for error in org.openehr.am.validation.RMInspector

### ISSUE THREAD

**Issue by vinyciaware** [openEHR/java-libs#24]
- In lines 327 and 357 of org.openehr.am.validation.RMInspector, a null pointer exception is raised since the "rmClass.getSimpleName()" (inside log.debug) is null. Therefore, we propose the insertion of the following code:

```
if(rmClass != null) {
     log.debug("rmClass: "+ rmClass.getSimpleName());
} else {
     log.debug("rmClass null");
}
```

**Comment by ppazos** [comment]
@vinyciaware IMO if rmClass is null, that is a bug and should be solved. it shouldn't be catched to be avoided.

**Comment by vinyciaware** [comment]
@ppazos This is true when the archetype is published. However, when we are validating a wrong archetype through org.openehr.am.validation.ArchetypeValidator which calls org.openehr.am.validation.RMInspector with no "simpleName" there is a problem. So, in this condition rmClass could be null.

**Comment by ppazos** [comment]
I think the solution is not to catch that, but to check the condition and throw a RMClassNotFound exception or something like that, since that is the real error: the archetype might contain a type that is not valid in the openEHR IM.

BTW, can you publish the archetype that is failing?
